src/components/calcite-input/calcite-input.e2e.ts

```diff
@@ -743,33 +744,55 @@ describe("calcite-input", () => {
     expect(calciteInputInput).not.toHaveReceivedEvent();
   });
 
-  it("should emit event when up or down clicked on input", async () => {
+  it("should emit an event every 500ms on mousedown on up/down buttons and stop on mouseup/mouseleave", async () => {
```

Comment:
The input is giving a very long precision on decimal steps, which is a pre existing issue, as it also lives on the master. The existing click test for checking incrementing with a decimal value doesn’t catch this long value, as it only occasionally pops up and goes away. The mousedown test for decimal values catches that and is currently failing. Erik suggested I write a utility test that determines decimal places and sets it so that stepping matches the precision of the provided step. I want to isolate this in a separate issue and address it independently.

Reply:
> I want to isolate this in a separate issue and address it independently.

Sounds like a great plan.